ando/tools/generator/nwb2bidsgenerator.py

```diff
@@ -56,29 +68,33 @@ def bep_organize(dataset_path, output_path=None, move_nwb=False, re_write=True,
                     subject_label = f'sub-{sb.subject_id}'
                 else:
                     subject_label = f'sub-{sb.date_of_birth.strftime("%Y%m%dT%H%M")}'
-                if not participants_df['ParticipantID'].str.contains(subject_label).any():
+                if not participants_df['ParticipantID'].str.contains(
+                        subject_label).any():
                     participants_df.loc[len(participants_df.index)] = \
                         [sb.species, subject_label, sb.sex[0] if sb.sex is not None else None,
                          sb.date_of_birth, sb.age, sb.genotype, sb.weight]
             else:
                 subject_label = 'sub-none'
-                if not participants_df['ParticipantID'].str.contains(subject_label).any():
+                if not participants_df['ParticipantID'].str.contains(
+                        subject_label).any():
                     participants_df.loc[len(participants_df.index)] = \
                         [None, subject_label, None, None, None, None, None]
 
             # dataset info:
             if dataset_desc_json is None:
-                dataset_desc_json = dict(InstitutionName=nwbfile.institution, InstitutionalDepartmentName=nwbfile.lab,
-                                         Name='Electrophysiology', BIDSVersion='1.0.X',
-                                         Licence='CC BY 4.0',
-                                         Authors=[
-                                             list(nwbfile.experimenter) if nwbfile.experimenter is not None else None])
+                dataset_desc_json = dict(
+                    InstitutionName=nwbfile.institution, InstitutionalDepartmentName=nwbfile.lab,
+                    Name='Electrophysiology', BIDSVersion='1.0.X',
+                    Licence='CC BY 4.0',
+                    Authors=[
+                        list(nwbfile.experimenter) if nwbfile.experimenter is not None else None])
             # sessions info:
             subject_path = output_path/subject_label
             bep_sessions_path = subject_path/f'{subject_label}_sessions.tsv'
             if not bep_sessions_path.exists():
                 print(f'writing for subject: {subject_label}')
-                sessions_df = pd.DataFrame(columns=['session_id', '#_trials', 'comment'])
+                sessions_df = pd.DataFrame(
+                    columns=['session_id', '#_trials', 'comment'])
```

Comment:
So, does these columns are fixed for a nwb dataset ?

Reply:
yes, 
session_id is anyway a required column based on the [BEP spec doc](https://docs.google.com/document/d/1oG-C8T-dWPqfVzL2W8HO3elWK8NIh2cOCPssRGv23n0/edit?ts=6038e4f4#)
#_trials, comment are additional optional fields for which info can be found in the nwb file

Reply:
In line 112 the number of trials is handled in more detail and `None` is used if the info can not be found within the NWB file. I guess the validation step at the end will show if all essential infos were present in the original NWB file.